plugins/package-managers/gradle/src/main/kotlin/Gradle.kt

```diff
@@ -327,3 +326,24 @@ private fun getGradleProperties(): Map<String, String> =
         }
         ?.toMap()
         .orEmpty()
+
+/**
+ * Convert this [OrtRepository] to a [RemoteRepository] taking the known properties into account.
+ * TODO: Also handle snapshot policy.
+ */
+private fun OrtRepository.toRemoteRepository(): RemoteRepository {
+    val builder = RemoteRepository.Builder(url, "default", url)
+
+    username?.let { user ->
+        password?.let { pwd ->
```

Comment:
Nit: Prefer `also` over `let` if the return value is not needed.

Reply:
Changed to the `apply` style as proposed below.